packages/rx-utils/src/__tests__/create-handler.utils.spec.ts

```diff
@@ -1,42 +1,46 @@
 import { createHandler } from '../create-handler.utils';
 import { Endomorphism, Lazy } from 'fp-ts/lib/function';
-import { constVoid } from 'fp-ts/lib/function';
 
 interface VoidThunk<A> {
 	(a: A): void;
 }
-
 describe('create-handler.utils.ts', function() {
 	describe('createHandler', function() {
 		it('should not fail', function() {
 			const a = createHandler();
 			const af1: Lazy<void> = a.handle; // adds possibility to use handle as Lazy<void>
 			const af2: Endomorphism<void> = a.handle; // keeps possibility to use as Endomorphism<void>
 			a.handle();
-			a.handle(undefined);
-			a.handle(constVoid());
+			// a.handle(undefined); // error
```

Comment:
Possibly unnecessary comments?

Reply:
no, i left it to have possibility to uncomment it and check that it fails as it should

Reply:
comments are not checked :) 
either we should write correct test or remove it

Reply:
amended